packages/react-instantsearch/src/core/createInstantSearchManager.js

```diff
@@ -29,6 +29,9 @@ export default function createInstantSearchManager({
   helper.on('result', handleSearchSuccess);
   helper.on('error', handleSearchError);
 
+  let derivedHelpers = {};
```

Comment:
if derivedHelpers is always re-assigned then it's initial value is not important and we should remove it so that we do not expect it to be used as a cache. It's just a shared variable between getSearchParameters() and search().